### DISCUSSION TITLE
Recognizing email address when verifying academic status

### DISCUSSION THREAD

**Discussion by gaomy3832** [community/community#58126] [GitHub Education]
### Hi everyone,

When I am trying to (re)verify my academic status (I am a faculty member) to get the education benefits, the system gives out some messages that make me wonder if the system can correctly recognize my email address. I have tried multiple times to verify but all failed. See #54126 and #51810. There are two issues as shown below.

First, my university (Tsinghua University, Beijing, China) uses different suffix for faculty and for students. Faculty get @mail.tsinghua.edu.cn or simply @tsinghua.edu.cn, while students get @mails.tsinghua.edu.cn. The system seems to only recognize the student emails. Please add the other two suffixes if possible.

Second, the system says my email is already used to get discount, but this is exactly the same account that I am trying to reverify and get back the benefits. I am not sure if this message is fine, or it is a mistake that prevents me from successfully passing the verification.

![Screen Shot 2023-06-15 at 5 28 28 PM](https://github.com/community/community/assets/3386287/10d4db3e-e278-4c47-99d7-e0ce708e5493)

**Comment by samus-aran** [comment]
Hi @gaomy3832 

Apologies on the delay. We've updated the school details and we do accept the @tsinghua.edu.cn domains (this is what the internal team have reviewed). The message is directed to students so you should be able to proceed with either email.

Regarding to your email that is associated to another account, this would require the support team to step in, specifically Billing & Account.   So please do contact [Account or billing issues](https://support.github.com/request) Support and they will explain the details of this message.

**Comment by gaomy3832** [comment]
Hi @samus-aran 

Thanks for the help and the investigation. However when I tried today, the verification still fails! Below is the attached screenshot for the message, which again contains no useful information about why. Could you please check again?


![Screen Shot 2023-08-07 at 11 33 26 AM](https://github.com/community/community/assets/3386287/d55c4e44-3871-4288-b079-5046f0632105)
